### DISCUSSION TITLE
Thank you for this Plug-in!!!!

### DISCUSSION THREAD

**Discussion by FelipeRearden** [jglev/obsidian-apply-patterns-plugin#3] [General]
Hello @publicus 

First of all, I wanna say thanks for creating this amazing plug-in. RegEx is something that I miss in Obsidian and now we have a powerful plug-in to integrate RegEx with Obsidian.

Your plug-in is extremely important in many of my workflows in Obsidian. This way, I have some Feature Request to share with you. 

I’m gonna separate them in different discussions to be easier to explain:
- [New Option to --> Apply Patterns: Apply Pattern to the whole Note](https://github.com/publicus/obsidian-apply-patterns-plugin/discussions/4)
- [Hotkeys for Specific Patterns](https://github.com/publicus/obsidian-apply-patterns-plugin/discussions/5)
- [Apply Patterns options INSIDE each Specific Pattern](https://github.com/publicus/obsidian-apply-patterns-plugin/discussions/8)
- [New Rule: Find AND Select All Matches](https://github.com/publicus/obsidian-apply-patterns-plugin/discussions/9)

These 4 feature requests are all related to each other.

Thanks :)

**Comment by jglev** [comment]
Thank you for your kind words, @FelipeRearden! I appreciated them! I also appreciate you breaking out each of your feature suggestions, and will respond to each shortly.

**Reply by FelipeRearden** [reply]
Hello @publicus !!!!

Thanks for the fast reply! 

I'm glad that you like my requests! I'm gonna read all of your replies.


Thanks for your attention and welcome to our community!!! :)

**Comment by javiavid** [comment]
I just wanted to thank you, for me it is also very useful to be able to make several consecutive regex to adapt my texts.

**Reply by jglev** [reply]
Thank you very much for your feedback!
